config/convert_rosbag2_to_non_annotated_t4_drs.yaml

```diff
@@ -0,0 +1,43 @@
+task: convert_rosbag2_to_non_annotated_t4
```

Comment:
Can we remove [config/convert_comlops_rosbag2_to_non_annotated_t4_sample.yaml](https://github.com/tier4/tier4_perception_dataset/blob/main/config/convert_comlops_rosbag2_to_non_annotated_t4_sample.yaml)?

Reply:
Sure. Since it's better to refer to `convert_rosbag2_to_non_annotated_t4_drs.yaml`, I will remove the sample yaml file.

Reply:
https://github.com/tier4/tier4_perception_dataset/pull/177/commits/7c485e40399311c84cdd338fdc617d8165914fcd